app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,31 @@
+class GroupsController < ApplicationController
+  load_and_authorize_resource :groups
+
+  def index
+    @groups = Group.all.where(user: current_user).includes(:expenses)
+  end
+
+  def new
+    @group = Group.new
+  end
+
+  def create
+    @group = Group.new(group_params)
+
+    respond_to do |format|
+      if @group.save
+        format.html { redirect_to group_url(@group), notice: 'Group was successfully created.' }
+        format.json { render :show, status: :created, location: @group }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @group.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
###  "Add a new category" page

- [ ] According to the project requirements, When the user clicks (or taps) the "Save" button to create the new category, it must take the user to the home page (category) on success and not stay on the add Category 'groups/new` page. Kindly correct this as well. ✍🏻